integration_tests/tests/integrity/integrity_line_item_enhanced.sql

```diff
@@ -0,0 +1,21 @@
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+-- this test is to make sure there is no fanout between the spine and the daily_overview
```

Comment:
Is this comment valid for this model? We don't use a spine anywhere when generating the line item enhanced standardized model.

Reply:
oops forgot to remove that from my paste